tree/main.go

```diff
@@ -75,3 +77,14 @@ func visit(path, indent string) (dirs, files int, err error) {
 	}
 	return dirs + 1, files, nil
 }
+
+func removeHidden(files []string) []string {
+	clean := []string{}
+	for _, f := range files {
+		if strings.HasPrefix(f, ".") {
```

Comment:
maybe use

```go
if f[0] != '.' {
   clean = append(clean, f)
}
```

that way the code is shorter and faster (`HasPrefix` is overkill here)